docs/oas/business.yaml

```diff
@@ -2809,7 +2809,7 @@ paths:
                         comments: []
                         date: '2024-07-25'
                         deletionLocked: false
-                        effectiveDate: '2024-07-25'
+                        effectiveDate: 2024-07-25T19:54:02.949764+00:00
```

Comment:
you will need the quotation for that, since it's a string type `'2024-07-25T19:54:02.949764+00:00'`